assets/js/components/event/PresenterStreams.tsx

```diff
@@ -12,10 +17,15 @@ const playerCallbacks: { [key: string]: (sourceType: SourceType) => void } = {};
 let webrtc: MembraneWebRTC | null = null;
 let webrtcConnecting = false;
 
-const includeKey = (storage: Presenter[], key: string): boolean => {
+const doesIncludeKey = (storage: Presenter[], key: string): boolean => {
   return storage.some((e) => e.email === key);
 };
 
+const useRerender = () => {
+  const [value, setValue] = useState(0);
+  return () => setValue(value + 1);
```

Comment:
this function is not component specific so it could be moved to some .ts file

Reply:
ok, I'll move it to some file in the `utils` directory.